### DISCUSSION TITLE
hyphens support

### DISCUSSION THREAD

**Discussion by JakobBruening** [gotenberg/gotenberg#944] [Q&A]
Is there any solution for use css `hyphens: auto;` without using external javascript libraries?

**Comment by NiklasBr** [comment]
A guess here: You might need a dictionary for the language to be hyphenated, see support at https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens#browser_compatibility

I think Firefox uses Hunspell which has a lot of l10n dictionaries, I suppose Chromium uses the same.

**Reply by JakobBruening** [reply]
I don't think this would work. Gotenberg is generally not compatible with the CSS setting, I think they would have to do so to make it working.

**Reply by NiklasBr** [reply]
Isn't that up to the renderer (i.e. Chromium or LibreOffice)?

**Reply by JakobBruening** [reply]
> Isn't that up to the renderer (i.e. Chromium or LibreOffice)?

In chromium, is does not work at least. And in libreoffice I experienced many other issues, which makes Chromium much more usable for my case.

**Reply by NiklasBr** [reply]
Still looks like a Chromium issue to me: [Bug: 353304848 – Hyphens are not rendered correctly using Chrome for Testing in headless mode](https://issues.chromium.org/issues/353304848).

**Comment by shox404** [comment]
Using hyphens: auto is a straightforward and effective way to manage text wrapping and improve readability without needing additional libraries.

**Reply by JakobBruening** [reply]
> Using hyphens: auto is a straightforward and effective way to manage text wrapping and improve readability without needing additional libraries.

Sure. But it is not working with gotenberg. In my html, it is working fine. But if I try to convert to pdf with gotenberg, it is not working. Already reported [here](https://github.com/gotenberg/gotenberg/issues/422)
